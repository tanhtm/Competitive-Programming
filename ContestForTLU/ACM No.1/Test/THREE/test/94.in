574765373000000000
