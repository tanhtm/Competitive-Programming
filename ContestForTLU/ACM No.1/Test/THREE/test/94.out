0 3 7 8 10 12 16 21 28 29 30 35 
1 2 4 9 11 14 15 18 23 24 25 27 31 32 33 34 36 37 
